Library/PagedContainer/PagedContainerViewModel.swift

```diff
@@ -0,0 +1,48 @@
+import Combine
+import Foundation
+import UIKit
+
+public class PagedContainerViewModel {
+  // Internal
+  private var subscriptions = Set<AnyCancellable>()
+
+  init() {
+    self.pageTitles = self.configureWithChildrenSubject.map { controllers in
+      controllers.map { $0.title ?? "Page " }
+    }.eraseToAnyPublisher()
+
+    self.displayChildViewControllerAtIndex = Publishers.CombineLatest(
```

Comment:
I feel like this should really be a `takeWhen` but I can't find the Combine equivalent. We want the index + children when the index changes.

Reply:
I don't think there is an equivalent? I don't think it matters for this one since I doubt the other signal can change. But if we really end up needing a `takeWhen` we may need to build our own. (Or, alternatively, store more state so that we don't need to grab everything from publishers?)

Reply:
Fair point that the other signal (probably) won't ever change! The only situation I could see would be if we ever added a new page of content to this tab, and that page of content was affected by logged in/logged out state.